src/main/java/MissingEventDurationException.java

```diff
@@ -0,0 +1,6 @@
+public class MissingEventDurationException extends Exception {
+    public MissingEventDurationException() {
+        super(" oops! event duration cannot be empty! please enter a duration after the task type and task name." +
+                "for example:\n      event cs2101 project meeting /from 4pm /to 7pm");
+    }
+}
```

Comment:
I really like how you specified different types of Exception and refactored them into different classes when the instructions didn't explicitly ask for it! It really demonstrated how you kept OOP principle in mind when designing your solution:)